### ISSUE TITLE
Missing Dockerfile for Alpine Linux

### ISSUE THREAD

**Issue by dirkmoors** [DataDog/datadog-agent#1788]
Hello,

In order to be able to use the Datadog Agent in an IoT environment, I need to get it running on Alpine Linux. There could be 2 approaches imho: either by having Datadog Agent Alpine base images, or by making the install-script or build-form-source-script work on Alpine Linux, so I can include it myself.

At the moment, I am limited to pushing my own metrics via the Python SDK, but I'm missing all the system metrics, let alone the process-metrics. 

Looking forward to having Alpine Support!
Thanks in advance!

**Comment by c0ze** [comment]
Yes for alpine support !

**Comment by flushentitypacket** [comment]
Is this what you're looking for?
https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/dogstatsd/alpine

**Comment by juanmrad** [comment]
Yes please can we have an alpine install script instead. I need to be able to install it on my custom alpine base docker images.

**Comment by kamatama41** [comment]
👍  for Alpine based datadog-agent Docker image
We are using the agent on an IoT environment which has low bandwidth. Current image size is very huge for us to pull.

**Comment by at-wat** [comment]
I made a Dockerfile and patches to build datadog agent v7 on Alpine, and it seems working on my use case.
https://github.com/seqsense/datadog-agent-alpine
I'll create PRs to apply the patches to fix compatibility with musl-libc and Alpine environment.
